spec/helpers/snappybara.rb

```diff
@@ -71,16 +71,16 @@ def self.wait_for(condition_name, max_wait_time: 30, polling_interval: 0.01)
       driver = example.metadata[:driver] if example.metadata[:driver]
 
       if example.metadata[:reset] || driver != Capybara.current_driver
```

Comment:
do we have any `reset` tags still in the tests? seems like this whole block could be deleted.

also remove commented out code